cmd/keeper/keeper.go

```diff
@@ -115,6 +117,7 @@ func init() {
 	cmdKeeper.PersistentFlags().StringVar(&cfg.pgSUUsername, "pg-su-username", user, "postgres superuser user name. Used for keeper managed instance access and pg_rewind based synchronization. It'll be created on db initialization. Defaults to the name of the effective user running stolon-keeper. Must be the same for all keepers.")
 	cmdKeeper.PersistentFlags().StringVar(&cfg.pgSUPassword, "pg-su-password", "", "postgres superuser password. Only one of --pg-su-password or --pg-su-passwordfile must be provided. Must be the same for all keepers.")
 	cmdKeeper.PersistentFlags().StringVar(&cfg.pgSUPasswordFile, "pg-su-passwordfile", "", "postgres superuser password file. Only one of --pg-su-password or --pg-su-passwordfile must be provided. Must be the same for all keepers)")
+	cmdKeeper.PersistentFlags().BoolVar(&cfg.logColor, "log-color", false, "enable color in log output (default if attached to a terminal)")
```

Comment:
Perhaps we could move this options and related logic inside the common command config: https://github.com/sorintlab/stolon/blob/master/cmd/common.go to avoid replicating it.